app/models/wiki.rb

```diff
@@ -104,6 +104,10 @@ def ensure_wiki_exists
     raise InvalidWikiError unless base_url == "https://#{servername}"
   end
 
+  def edits_enabled?
+    ENV["edit_#{domain}"]
```

Comment:
This will return a truthy value if it is set to "false" in application.yml. We should follow the convention used elsewhere in the app for config switches: `ENV["edit_#{domain}"] == 'true'. That way, it always returns true or false (as a `?` method should) and only returns true if it's unambiguously set to 'true'.